temporian/core/operators/glue.py

```diff
@@ -208,3 +208,21 @@ def glue(
     inputs_dict = {f"input_{idx}": input for idx, input in enumerate(inputs)}
 
     return GlueOperator(**inputs_dict).outputs["output"]  # type: ignore
+
+
+@typecheck
+@compile
+def assign(input: EventSetOrNode, **others: EventSetOrNode) -> EventSetOrNode:
+    assert isinstance(input, EventSetNode)
+    others_renamed = []
+    for name, other in others.items():
+        if len(other.schema.features) != 1:
+            raise ValueError(
+                "Cannot assign an EventSets with multiple features"
```

Comment:
typo, an EventSet. but would change to: The assigned EventSets must have a single feature (since it also covers the case of it having 0)